installation/local/bootstrap.sh

```diff
@@ -64,79 +84,75 @@ echo "1" | sudo update-alternatives --config mpi 1>/dev/null 2>&1 # --> choosing
 echo "1" | sudo update-alternatives --config mpirun 1>/dev/null 2>&1 # --> choosing mpirun
 
 #
-# STEP 3 - install NEST Desktop (and dependencies for NEST Server)
-#
-pip install nest-desktop==3.3.0a1 \
-	flask flask-cors gunicorn
-
-#
-# STEP 4 - TVB
+# STEP 3 - Install Python packages of TVB, NEST Desktop (and dependencies for NEST Server)
 #
 # NOTE: Specific versions are required for some packages
 pip install --no-cache --target=${CO_SIM_SITE_PACKAGES} \
```

Comment:
Including the additions from Kim and Muhammad:

```suggestion
pip install --no-cache --upgrade --target=${CO_SIM_SITE_PACKAGES} \
```
But please have a final check/look at it.